pkg/cli/admin/upgrade/recommend/examples/4.12.16-longest-not-recommended.output

```diff
@@ -24,6 +24,6 @@ And 45 older 4.13 updates you can see with '--show-outdated-releases' or '--vers
 
 Updates to 4.12:
   VERSION     ISSUES
-  4.12.64     
-  4.12.63     
+  4.12.64     no known issues
+  4.12.63     no known issues
```

Comment:
Currently the semantics for the previously-empty entries was something like:

> As of this cluster's last poll of its `upstream` update service, there may or may not have been any known risks declared by that update service.  If there were risks, we've successfully evaluated all of them and are confident that they don't match this cluster.

That's obviously too much for a table entry, but I'm slightly concerned that if we say `no known issues` but the admin is aware of a risk affecting that update in general, they'll think "what? _I_ know there's a risk" and forget about the CVO's "am I actually exposed to this one?" PromQL evaluation.

But I'm not so concerned about that that I'm sure `no known issues` is too short.  Just concerned enough to float it here for more feedback.  If you think `no known issues` is likely going to be fine, or folks don't have alternatives they're more confident about soon, I am completely fine going with `no known issues`.  We can always revisit in future pull requests, if we hear about admins getting confused.

Reply:
How about "no known issues affecting this cluster", or "no affecting issues"?
My intension is to relief the concern, e.g., `4.12.64` may have risks but none of them affects this cluster.

Reply:
I've pushed c33f996d7 -> aaa6d774a to pivot to `no known issues relevant to this cluster` to match the existing:

```console
$ git --no-pager grep 'no known issues' origin/master
origin/master:pkg/cli/admin/upgrade/recommend/recommend.go:                                             fmt.Fprintf(o.Out, "Update to %s has no known issues relevant to this cluster.\nImage: %s\nURL: %s\n", update.Release.Version, update.Release.Image, update.Release.URL)
```

Reply:
Thanks.